src/annotator/test/guest-test.js

```diff
@@ -29,10 +29,6 @@ class FakeTextRange {
   }
 }
 
-// A little helper which returns a promise that resolves after a timeout
-const timeoutPromise = (millis = 0) =>
-  new Promise(resolve => setTimeout(resolve, millis));
-
```

Comment:
This is no longer needed because tests can explicitly await `createAnnotation`'s results.